### DISCUSSION TITLE
Many-to-one Relationship

### DISCUSSION THREAD

**Discussion by technotip** [prisma/prisma#16609] [Q&A] [answered]
I'm using Prisma + MongoDB.

I've 2 collections User and Post:

```
model User {
  userId   String    @id @default(auto()) @map("_id") @db.ObjectId
  name    String
}

model Post {
  postId     String    @id @default(auto()) @map("_id") @db.ObjectId
  favorited String[]
}
```

I'm simplifying the schema to avoid any confusion. (I've not included any relationship attributes to avoid confusion.)

I want to store userIds in **favorited** field. And then retrieve user details using those userIds present in favorited field.

One user can favorite any number of posts.

If I store postId in User collection, the array keeps growing. So I don't want to do that. Instead, I want to store the userIds in favorited field in Post collection. But so far I've not found a way of doing it. Kindly share your solution.

**Comment by nurul3101** [comment, marked as answer]
Just for reference, this question is related to this Feature Request
- #12725 

MongoDB one way embedding isn't supported yet, there are a few alternatives which are suggested in the Feature Request.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
